pylintrc

```diff
@@ -324,7 +324,7 @@ dummy-variables-rgx=_+$|(_[a-zA-Z0-9_]*[a-zA-Z0-9]+?$)|dummy|^ignored_|^unused_|
 
 # Argument names that match this expression will be ignored. Default to name
 # with leading underscore.
-ignored-argument-names=_.*|^ignored_|^unused_
+ignored-argument-names=_.*|^ignored_|^unused_|^kwargs|^args|^mock_.+
```

Comment:
I think we should separate it out to separate (very small) PR. It's a good change, but it should be separated out.

Reply:
This is part of: https://github.com/apache/airflow/pull/5586
Can you look at this?

Reply:
Perfect! -> Approved (but let's give a chance to @BasPH to take a look).

Reply:
Other PR LGTM, I merged it 👍

Reply:
Nice! I have rebase this PR to use the change from https://github.com/apache/airflow/pull/5586.